src/chocolatey.install/chocolatey.wxs

```diff
@@ -54,14 +46,6 @@
         <Directory Id="INSTALLDIR" Name="chocolatey">
           <Component Id="ChocolateyInstallDirectory" Guid="{9E538CD6-6F07-43E5-B8E2-ED31995F3D1C}" Win64="no">
             <CreateFolder />
-            <RemoveFile Id="ChocoEXE" On="uninstall" Name="choco.exe" />
-            <RemoveFile Id="ChocoIgnore" On="uninstall" Name="choco.exe.ignore" />
-            <RemoveFile Id="ChocoManifest" On="uninstall" Name="choco.exe.manifest" />
-            <RemoveFile Id="ChocoCredits" On="uninstall" Name="CREDITS.txt" />
-            <RemoveFile Id="ChocoLicense" On="uninstall" Name="LICENSE.txt" />
-            <util:RemoveFolderEx Id="HelpersDirectory" On="uninstall" Property="HelpersDirectoryPath" />
-            <util:RemoveFolderEx Id="RedirectsDirectory" On="uninstall" Property="RedirectsDirectoryPath" />
-            <util:RemoveFolderEx Id="ToolsDirectory" On="uninstall" Property="ToolsDirectoryPath" />
```

Comment:
It should be a no-op. The goal is to remove from Programs and Features but not actually remove Chocolatey itself. We should notify the user to that @corbob ?

Reply:
Yeah, in this version we're going to make it a no-op since the previous behaviour doesn't actually remove all of Chocolatey, and instead leaves it in a broken state that doesn't allow reinstallation without manual intervention. I'll be doing up an ADR for this with the reasoning and options considered. The main reason to do a no-op right now is we would rather have a non operative uninstall than one that isn't working and don't want to rush the work of having a proper uninstall.

With regards to notifying the user @pauby, I wasn't finding anything in my searching about changing things, but I just noticed some things in this diff that I think can work, so I'll look into that.